Source/WebKit/Shared/ApplePay/cocoa/WebPaymentCoordinatorProxyCocoa.mm

```diff
@@ -390,6 +390,16 @@ static PKPaymentRequestAPIType toAPIType(WebCore::ApplePaySessionPaymentRequest:
         [result setDeferredPaymentRequest:platformDeferredPaymentRequest(*deferredPaymentRequest).get()];
 #endif
 
+#if HAVE(PASSKIT_USER_AGENT)
+    auto userAgent = [webPageProxyID] {
+        if (auto page = WebProcessProxy::webPage(webPageProxyID))
+            return page->userAgent();
+        return WebPageProxy::standardUserAgent();
+    }();
+    if ([result respondsToSelector:@selector(setUserAgent:)])
+        [result setUserAgent:userAgent];
```

Comment:
I'd drop in the same `FIXME` comment that you have above, so that we remove the selector check when we remove the declaration.

Reply:
Added the FIXME comment in https://github.com/WebKit/WebKit/commit/f5b0870f54f7f3f2669d787c2bcb8e348149dda2